modular_skyrat/modules/emotes/code/emotes.dm

```diff
@@ -55,6 +57,8 @@
 /datum/emote/living/sneeze/get_sound(mob/living/user)
 	if(isvox(user))
 		return 'modular_skyrat/modules/emotes/sound/emotes/voxsneeze.ogg'
+	if(isnabber(user))
+		return 'modular_skyrat/modules/emotes/sound/emotes/nabbersneeze.ogg'
```

Comment:
you need to source the audio here and make sure it's either 100% original or CC-BY-NC-SA 2.0